k8s/helm/templates/deployment.yaml

```diff
@@ -121,3 +121,5 @@ spec:
               value: "{{- .Values.newRelic.licenseKey }}"
             - name: NEW_RELIC_APP_NAME
               value: "{{- .Values.newRelic.appName }}"
+            - name: CONFIRMATION_EMAIL_REQUEST_LIMIT
+              value: "{{- .Values.confirmationEmailRequestLimit }}"
```

Comment:
There should be a default in `values.yaml`, so that this only needs to be provided for environments that need to change it.